Makefile

```diff
@@ -3,29 +3,29 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
```

Comment:
what about adding:-
`PATH := $(GOPATH)bin:$(PATH)`
`export PATH`

at the top of the file, then gom *should be visible in the path?

Otherwise users of this will need to run : go get github.com/mattn/gom , on their own .. what do you think?